kiva/tests/test_marker_rendering.py

```diff
@@ -0,0 +1,69 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+import contextlib
+import unittest
+
+import numpy as np
+
+from kiva import constants
+from kiva.marker_renderer import MarkerRendererRGB24
+
+
+class TestMarkerDrawing(unittest.TestCase):
+    def setUp(self):
+        self.buffer = np.ones((300, 300, 3), dtype=np.uint8)
+        self.gc = MarkerRendererRGB24(self.buffer)
```

Comment:
Might be a good idea to test all variations of the renderer here?

Reply:
that's definitely a good idea!